bin/ParseDarwinSamplesheet.sh

```diff
@@ -138,13 +144,45 @@ fetch_data () {
 			else
 				_fileType='BAM'
 			fi
+		elif [[ ! -d "${_searchPath[0]}" ]]
```

Comment:
dit werkt 100% zeker? Want er zitten in je searchPath meerdere sterretjes, een `-f ` struikelt hier bijvoorbeeld wel over

Reply:
jazeker, heel vaak getest.